pyupgrade.py

```diff
@@ -2429,8 +2447,11 @@ def _fix_py36_plus(contents_text: str) -> str:
             if tokens[end].line != token.line:
                 continue
 
-            tokens[i] = token._replace(src=_to_fstring(token.src, node))
-            del tokens[i + 1:end + 1]
+            try:
+                tokens[i] = token._replace(src=_to_fstring(token.src, node))
+                del tokens[i + 1:end + 1]
+            except KeyError:
+                pass
```

Comment:
I was looking for an unparse-like function that could be used in the `_to_fstring` because this feels like it's handling an implementation detail.